### PR TITLE
cleanup + fix backticks in readme

### PR DESCRIPTION
**Pull request by rethab**
the `import` statement actually needs backticks after `$ivy.` so I added them. however using them with each snipped is somewhat confusing since they are already formatted correctly by indenting and in the case of the import they are actually needed. hopefully this makes it clearer and copy-paste ready

### CODE REVIEW

File: contrib/bsp/readme.md [hunk 1]
```diff
@@ -30,11 +30,11 @@ After the bsp support module would be published, it should be enough to:
 1) Install mill
 2) Add the following import statement in the build.sc of your project:
 
-        `import $ivy.com.lihaoyi::mill-contrib-bsp:$OFFICIAL_MILL_VERSION`
+        import $ivy.`com.lihaoyi::mill-contrib-bsp:$OFFICIAL_MILL_VERSION`
```
Comment (lefou) [inline contrib/bsp/readme.md:33]:
```suggestion
        import $ivy.`com.lihaoyi::mill-contrib-bsp:$MILL_VERSION`
```

This will then work out-of-the-box.

**Comment by rethab** [comment]
Totally, linking to the docs would be better. Or just not have a readme at all. I think I just instinctively looked for a RAEDME and would have looked at the docs instead so I think a README is not necessary.

**Comment by lefou** [comment]
Agreed. Can you change your PR to delete the README?

**Comment by rethab** [comment]
Done @lefou

**Comment by lefou** [comment]
Thanks!

**Pull request merged**